pkg/types/aws/validation/machinepool.go

```diff
@@ -108,6 +109,25 @@ func validateIOPS(p *aws.MachinePool, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
+func validateThroughput(p *aws.MachinePool, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	volumeType := strings.ToLower(p.EC2RootVolume.Type)
+	throughput := p.EC2RootVolume.Throughput
+
+	switch volumeType {
+	case "gp3":
+		if throughput < 125 || throughput > 2000 {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("throughput"), throughput, "throughput must be between 125 MiB/s and 1000 MiB/s"))
```

Comment:
```suggestion
			allErrs = append(allErrs, field.Invalid(fldPath.Child("throughput"), throughput, "throughput must be between 125 MiB/s and 2000 MiB/s"))
```

Max range is now 2000 MiB/s :grin:

Reply:
Aha! Fixed.